core/src/core/pmmr/backend.rs

```diff
@@ -39,6 +39,9 @@ pub trait Backend<T: PMMRable> {
 	/// Get a Hash by insertion position.
 	fn get_hash(&self, position: u64) -> Option<Hash>;
 
+	/// Get a leaf hash by insertion position, ignoring the leaf set
```

Comment:
This should not be necessary - we already have `get_from_file()` below, which does exactly this.